hikari/guilds.py

```diff
@@ -183,6 +183,34 @@ class GuildFeature(str, enums.Enum):
     ROLE_SUBSCRIPTIONS_AVAILABLE_FOR_PURCHASE = "ROLE_SUBSCRIPTIONS_AVAILABLE_FOR_PURCHASE"
     """Guild has role subscriptions available for purchase."""
 
+    INVITES_DISABLED = "INVITES_DISABLED"
+    """Guild has paused invites, preventing new users from joining."""
+
+    RAID_ALERTS_DISABLED = "RAID_ALERTS_DISABLED"
+    """Guild has disabled alerts for join raids in the configured safety alerts channel."""
+
+
+@typing.final
+class MutableGuildFeature(str, enums.Enum):
```

Comment:
I don't *hate* this, because it makes it type-safe, but as you said, it makes it less flexible and is generally not consistent with how hikari handles these things. Usually these things are not checked and is up to the user to make sure/build abstractions that check on top of hikari.

Reply:
I'm not sure of this myself. I originally thought that having a separate enum would be better, but had found myself fighting with PyCharm over `GuildFeature` vs `MutableGuildFeature`. Another feat I tried was to have `MutableGuildFeature` reference `GuildFeature` directly, but `pytest` wasn't happy with it, spitting `TypeError: Can't intern (something something I forgot)`